sql/src/main/scala/org/apache/spark/sql/sedona_sql/expressions/Functions.scala

```diff
@@ -727,6 +728,88 @@ case class ST_MinimumBoundingCircle(inputExpressions: Seq[Expression])
 }
 
 
+/**
+ * Return a linestring being a substring of the input one starting and ending at the given fractions of total 2d length.
+ * Second and third arguments are Double values between 0 and 1. This only works with LINESTRINGs.
+ *
+ * @param inputExpressions
+ */
+case class ST_LineSubString(inputExpressions: Seq[Expression])
```

Comment:
Thank you for your contribution! Not sure whether or not Spark function name is case sensitive or not. So would you please change the name to 'ST_LineSubstring' to make it consistent with PostGIS? https://postgis.net/docs/ST_LineSubstring.html

Reply:
Yes. You are right. It would be better to be consistent with PostGIS.